### DISCUSSION TITLE
Serverless Function (Via API route) responds with the built code file, instead of executing it

### DISCUSSION THREAD

**Discussion by scroobius-pip** [vercel/community#90] [Help] [answered]
I created two deployments, exactly the same config. But one responds with the code file instead of executing it.

Here's my project structure:
<img width="157" alt="image" src="https://user-images.githubusercontent.com/11554364/149141340-6c450f22-13ee-4b8c-bb57-5b5f34f1c676.png">

My now.json file looks like this:

{ 
    "alias": "api.rade.trade",
    "version": 2,
    "functions":{
        "api/index.js":{
            "runtime":"@vercel/node@1.12.1"
        }
    },
    "routes": [
        {
            "src": "/.*",
            "dest": "api/index.js"
        }
    ]
}

Here's the endpoints to the two deployments:
This one works properly:
https://rade-api-6yzyys1hq-simdi-jinkins.vercel.app
This one works improperly:
https://rade-api-gyfolz1ix-simdi-jinkins.vercel.app

It's very frustrating because i can't reproduce the issue

**Comment by RizBizKits** [comment]
Hey! I'm noting the same result on both links (i.e. `404: NOT_FOUND`). Are you able to double-check if your 'works properly' example deployment that you've shared is functioning as expected?

**Comment by Pieparker** [comment, marked as answer]
This conversation appears to have gone quiet. To keep discussions on topic and to prevent the resurrection of old topics, this discussion has been closed. If issues persist or a future reader stumbles upon this discussion and is exhibiting the same problem, please create a new discussion.
